content/_policy/index._en.md

```diff
@@ -9,6 +9,7 @@ redirect_from:
   - /docs/Logingov_PIA_August2019.pdf/
   - /docs/Privacy-Impact-Assessment 9-18.pdf/
   - /docs/privacy-impact-assessment.pdf/
+  - /docs/accessibility-assessment-ial2-identity-verification-process.pdf/  
```

Comment:
This would redirect the old URL to `/policy/`, but we'd want it to redirect to `docs/accessibility-assessment-ial2-identity-verification-process.pdf`.

The end result is that this link should work and should download as the report document: https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/18f/identity-site/LG-8904-replace-accessibility-report-document/docs/accessibility-assessment-ial2-identity-verification-process.pdf

Locally I was able to get this working with a new Markdown file in `docs`:

```diff
diff --git a/_config.yml b/_config.yml
index 9b0a1dc3..49718543 100644
--- a/_config.yml
+++ b/_config.yml
@@ -101,2 +101,3 @@ include:
   - docs/accessibility-assessment-identity-verification-process.pdf
+  - docs/accessibility-assessment-ial2-identity-verification-process.pdf/index.md
 
diff --git a/content/_policy/index._en.md b/content/_policy/index._en.md
index 38ccb033..fe0adc61 100644
--- a/content/_policy/index._en.md
+++ b/content/_policy/index._en.md
@@ -11,3 +11,2 @@ redirect_from:
   - /docs/privacy-impact-assessment.pdf/
-  - /docs/accessibility-assessment-ial2-identity-verification-process.pdf/  
 ---
diff --git a/docs/accessibility-assessment-ial2-identity-verification-process.pdf/index.md b/docs/accessibility-assessment-ial2-identity-verification-process.pdf/index.md
new file mode 100644
index 00000000..d9a7e2b7
--- /dev/null
+++ b/docs/accessibility-assessment-ial2-identity-verification-process.pdf/index.md
@@ -0,0 +1,3 @@
+---
+redirect_to: /docs/accessibility-assessment-identity-verification-process.pdf
+---
```

Reply:
Nice, that works for me locally. Clever trick.

I had copied the older style. It's worth noting that the previous attempts at redirects, for pdfs, seen in the file `identity-site/content/_policy/index._en.md` do not seem to work locally or in production when you inspect what is retrieved.

Specifically the following redirects:
```
redirect_from:
  - /docs/Logingov_PIA_August2019.pdf/
  - /docs/Privacy-Impact-Assessment 9-18.pdf/
  - /docs/privacy-impact-assessment.pdf/
 ```

In my testing a direct request to those URLs will attempt to download the file, but the file is not a pdf.

Reply:
This hasn't worked for me in my attempts, along with some variations. At times I thought it worked.

It's also worth pointing out that the code in `identity-site/content/_policy/index._en.md`, which features similar file redirects, does not work appropriately. Direct hits to those URLs will download something, but not a PDF upon closer inspection.

```
redirect_from:
  - /docs/Logingov_PIA_August2019.pdf/
  - /docs/Privacy-Impact-Assessment 9-18.pdf/
  - /docs/privacy-impact-assessment.pdf/
```